cowprotocol/watch-tower
# fix: memory leak

# Description
This PR addresses potential sources of the memory leak that was observed in staging.

# Changes

- [ ] Remove `express-prometheus-middlware`
- [ ] Ensure flushing to `leveldb` on every block

## How to test
1. Deploy to staging infrastructure environment
2. Observe pod memory consumption

## Related Issues

Fixes #98
